config/filter.go

```diff
@@ -73,6 +97,7 @@ func (t *Config) startFilters() (err error) {
 				}
 			case event := <-t.chInFilter:
 				for _, filter := range filters {
+					event = filter.CommonFilter(t.ctx, event)
```

Comment:
I think this filter should be processed only after event filter success?

Reply:
Why? The behavior seems does not conflict.

Reply:
I think we only need action after filter success. Otherwise, we can use mutate filter do the same thing.

BTW, logstash's filters also do this on filter success:
https://www.elastic.co/guide/en/logstash/current/plugins-filters-date.html#plugins-filters-date-common-options

Reply:
So, I think we could change the `Event` function and return the `success bool` to inform `CommonFilter` to do so.

Reply:
I'm neutral about this, @horacimacias what do you think?

Reply:
For example, I need to parse string to date, and it may not success always, I should only remove the original value when it is successful.

Like what I did in #80 's example:
```
          - type: cond
            condition: "!empty([nginx.access.timestamp])"
            filter:
              - type: date
                format: ["UNIX"]
                source: "nginx.access.timestamp"
            else_filter:
              - type: date
                format: ["dd/MMM/YYYY:H:m:s Z"]
                joda: true
                source: "nginx.access.time"
          - type: cond
            condition: "!('gogstash_filter_date_error' IN map([tags]))"
            filter:
              - type: remove_field
                fields: ['nginx.access.timestamp', 'nginx.access.time']
```

If we do this common filter only if filter success, it could be simplified without loss original value.

Reply:
I think that makes sense, yes. Agreed